remake: doesn't build, multiply defined symbols

I get a bunch of multiply defined symbols:

```
/usr/lib/gcc/x86_64-pc-msys/10.2.0/../../../../x86_64-pc-msys/bin/ld: src/commands.o:commands.c:(.bss+0x8): multiple definition of `read_makefiles'; src/ar.o:ar.c:(.bss+0x0): first defined here
/usr/lib/gcc/x86_64-pc-msys/10.2.0/../../../../x86_64-pc-msys/bin/ld: src/commands.o:commands.c:(.bss+0x10): multiple definition of `stack_limit'; src/ar.o:ar.c:(.bss+0x10): first defined here
/usr/lib/gcc/x86_64-pc-msys/10.2.0/../../../../x86_64-pc-msys/bin/ld: src/debug.o:debug.c:(.bss+0x20): multiple definition of `stack_limit'; src/ar.o:ar.c:(.bss+0x10): first defined here
/usr/lib/gcc/x86_64-pc-msys/10.2.0/../../../../x86_64-pc-msys/bin/ld: src/default.o:default.c:(.bss+0x0): multiple definition of `read_makefiles'; src/ar.o:ar.c:(.bss+0x0): first defined here
...
```

https://github.com/jeremyd2019/MSYS2-packages/runs/1339997000?check_suite_focus=true#step:4:1250

--- a/remake/PKGBUILD
+++ b/remake/PKGBUILD
@@ -5,7 +5,7 @@ conflicts=("remake-git")
 replaces=("remake-git")
 _remakever="dbg-1.5"
 pkgver=4.3+dbg1.5
-pkgrel=1
+pkgrel=2
 pkgdesc="Enhanced GNU Make - tracing, error reporting, debugging, profiling and more"
 arch=('i686' 'x86_64')
 url="https://github.com/rocky/remake"
@@ -28,6 +28,7 @@ prepare() {
 
 build() {
   cd ${srcdir}/${pkgname}-4.3+${_remakever}
+  CFLAGS+=" -fcommon"
   ./configure \
     --prefix=/usr \
     --build=${CHOST} \

